src/astro/sql/__init__.py

```diff
@@ -31,6 +31,31 @@ def transform(
     handler: Optional[Callable] = None,
     **kwargs: Any,
 ) -> TaskDecorator:
+    """
+    Given a python function that returns a SQL statement and (optional) tables, execute the SQL statement and output
+    the result into a SQL table.
+
+    Use this function as a decorator like so:
+
+      @transform
+      def my_sql_statement(table1: Table, table2: Table) -> Table:
+          return "SELECT * FROM {{table1}} JOIN {{table2}}"
+
+    In this example, by identifying the parameters as `Table` objects, astro knows to automatically convert those
+    objects into tables (if they are, for example, a dataframe). Any type besides table will lead astro to assume
+    you do not want the parameter converted.
+
+    :param python_callable:
+    :param multiple_outputs:
+    :param conn_id:
+    :param parameters:
+    :param database:
+    :param schema:
+    :param handler:
+    :param kwargs:
```

Comment:
This PR should not have been merged with these changes!

This is how it looks like:

<img width="700" alt="image" src="https://user-images.githubusercontent.com/8811558/175362814-eda1b6d9-b27a-4e78-8fe2-1824c8a81e0e.png">

Reply:
https://astro-sdk.readthedocs.io/en/create-docs-ci/autoapi/astro/sql/index.html

Reply:
There needs to be a description for it

Reply:
@kaxil based on discussion with tatiana I was given the sense that this PR would be more about creating the infrastructure for a readthedocs site. I think that actually fixing the docs can be handled in subsequent tickets (As there are a lot of places we should be fixing the docs now that we can see them)